test/config/notafter-backfiller.json

```diff
@@ -0,0 +1,17 @@
+{
+  "NotAfterBackFiller": {
+    "dbConnectFile": "test/secrets/backfiller_dburl",
+    "maxDBConns": 10
+  },
+
+  "statsd": {
+    "server": "localhost:8125",
+    "prefix": "Boulder"
+  },
+
+  "syslog": {
+    "network": "",
```

Comment:
The syslog config object no longer has network or server fields, so leave these out. I'll send a PR to remove from existing ones.